data/json/scenarios.json

```diff
@@ -95,7 +95,8 @@
       "sloc_horse_ranch",
       "sloc_lighthouse_ground",
       "sloc_cabin_lake",
-      "sloc_freshwater_research_station"
+      "sloc_freshwater_research_station",
+      "light_industry_10_scen"
```

Comment:
No, I'm sorry I steered you wrong here and you had that right.  You are missing editing in the new Overmap Special since you've got new terrain and a new mapgen.